src/main/java/com/ss/ita/rozetka/ui/pages/HomePage.java

```diff
@@ -0,0 +1,18 @@
+package com.ss.ita.rozetka.ui.pages;
+
+import java.util.ArrayList;
+import java.util.List;
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Selenide.*;
+
+public class HomePage extends Header{
+    public ComputersNotebooksPage getComputersNotebooksPage(){
+         $x("(//a[@href = 'https://rozetka.com.ua/computers-notebooks/c80253/'])[2]").click();
```

Comment:
methods named get*() should only return data and don't do any page navigation actions